builder/bundles.go

```diff
@@ -588,7 +588,7 @@ func (b *Builder) buildBundles(set bundleSet) error {
 
 	// Write INI files. These are used to communicate to the next step of mixing (build update).
 	var serverINI bytes.Buffer
-	fmt.Fprintf(&serverINI, `[Server]
+	_, err = fmt.Fprintf(&serverINI, `[Server]
```

Comment:
Don't check this error. It is just a `bytes.Buffer` which will never fail, so you can just throw it out. See earlier discussion with @tmarcu